homeassistant/components/cloud/http_api.py

```diff
@@ -587,10 +590,16 @@ async def google_assistant_get(
         )
         return
 
+    assistant_options: Mapping[str, Any] = {}
+    with suppress(HomeAssistantError, KeyError):
+        settings = exposed_entities.async_get_entity_settings(hass, entity_id)
```

Comment:
Should this just return an empty dict if entity is not registered? KeyError feels a bit random.